vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/gce/gce_urlmap.go

```diff
@@ -67,9 +67,11 @@ func (gce *GCECloud) DeleteUrlMap(name string) error {
 }
 
 // ListUrlMaps lists all UrlMaps in the project.
-func (gce *GCECloud) ListUrlMaps() (*compute.UrlMapList, error) {
-	mc := newUrlMapMetricContext("list")
+func (gce *GCECloud) ListUrlMaps() ([]*compute.UrlMap, error) {
```

Comment:
Remove all these changes to this file?

Reply:
Moved all vendor/ changes to https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/pull/150.
They were in this PR for tests to pass. Rebased to remove these changes now that https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/pull/150 is merged.